wondrous-app/components/Common/NewInviteLinkModal/styles.tsx

```diff
@@ -0,0 +1,374 @@
+
+export const StyledSelect = styled(Select)`
+  && {
+    background: #0f0f0f;
+    width: 141px;
+    max-width: 100%;
+    height: 40px;
+    color: #fff;
+    border-radius: ${(props) => (props.rounded ? '6px' : '0')};
+    font-size: 14px;
+    text-transform: ${(props) => (props.capitalize ? 'capitalize' : 'inherit')};
+  }
+
+  & .MuiSelect-root {
+    padding-left: 12px;
+  }
+
+  & .MuiInputBase-root {
+    border-radius: 0 6px 6px 0;
+  }
+
+  & .MuiInput-underline {
+    :hover:not(.Mui-disabled)::before {
+      border: none;
+      ::before {
+        border: none;
+      }
+      ::before {
+        border: none;
+      }
+    }
+  }
+
+  & .MuiSvgIcon-root {
+    color: ${(props) => props.svgColor};
+  }
+`;
+
+export const InviteThruLinkSelect = styled(({ className, ...props }) => (
+  <StyledSelect
+    {...props}
+    capitalize={true}
+    svgColor="#CCBBFF"
+    rounded={true}
+    MenuProps={{ classes: { paper: className } }}
+  />
+))`
+  &.MuiPaper-root {
+    background: linear-gradient(180deg, #1e1e1e 0%, #141414 109.19%);
+    width: 141px;
+    color: #fff;
+  }
+  &.MuiPaper-root > .MuiList-padding {
+    padding: 0;
+  }
+`;
+
+export const InviteThruLinkMenuItem = styled(MenuItem)`
+  && {
+    background: #121212;
+    color: #c4c4c4;
+    margin: 6px;
+    border-radius: 6px;
+    font-size: 13px;
+    border: 1px solid transparent;
+    height: 34px;
+  }
+
+  &&:hover {
+    background: rgba(0, 0, 0, 1);
+    border: 1px solid #7427ff;
+  }
+
+  .MuiListItem-root.Mui-selected,
+  .MuiListItem-root.Mui-selected:hover {
+    background: none;
+  }
+`;
+
+export const InviteThruLinkButton = styled(Button)`
+  && {
+    border: 1px solid #353535;
+    background: #0f0f0f;
+    font-weight: 400;
+    width: 137px;
+    flex-basis: 137px;
+    height: 40px;
+    border-radius: 6px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 0 12px 0 0;
+    border-radius: 100px;
+
+    :hover {
+      background: #353535;
+    }
+
+    &::before {
+      content: '';
+      position: absolute;
+      inset: 0;
+      background: linear-gradient(270deg, #ccbbff -5.62%, #7427ff 45.92%, #00baff 103.12%);
+      -webkit-mask: linear-gradient(#fff 0 0) content-box, linear-gradient(#fff 0 0);
+      -webkit-mask-composite: xor;
+      mask-composite: exclude;
+      padding: 1px;
+      border-radius: 180px;
+    }
+  }
+`;
+
+export const InviteThruLinkButtonLabel = styled(Typography)`
+  && {
+    color: white;
+    font-size: 14px;
+    margin-left: 25px;
+    font-weight: 500;
+  }
+`;
+
+export const InviteThruLinkButtonSuccessLabel = styled(InviteThruLinkButtonLabel)`
+  && {
+    background: #ffffff;
+    background: -webkit-linear-gradient(to bottom, #ffffff 0%, #06ffa5 100%);
+    background: -moz-linear-gradient(to bottom, #ffffff 0%, #06ffa5 100%);
+    background: linear-gradient(to bottom, #ffffff 0%, #06ffa5 100%);
```

Comment:
You can use background without prefixes, the preprocessor will do all the needed things based on the supported browsers.